.github/dependabot.yml

```diff
@@ -17,35 +17,35 @@ updates:
       patterns:
       - "@embroider/*"
   ignore:
-  - dependency-name: "@babel/core"
```

Comment:
Not sure about this `yaml` spacing change. I think lists are supposed to be unindented.

Reply:
corrected :white_check_mark: